### ISSUE TITLE
How to use trained model from pipeline.sh to do prediction

### ISSUE THREAD

**Issue by Maxwell-downtown** [OATML-Markslab/ProteinNPT#8]
Thanks again for your answer in https://github.com/OATML-Markslab/ProteinNPT/issues/7,
I still got one more small question. So, after the model is trained on certain assays with this `pipeline.sh` script, is it possible to use the trained model to test other mutations of the same protein that aren't in the training assay and give a prediction value? I tried to go through `pipeline.py` and the relating packages, but haven't figured out the right way to call the model for testing.

**Comment by pascalnotin** [comment]
Hi @Maxwell-downtown - we will add example scripts for this in the future (have to load in the checkpoint saved at the end of training at `$DATA_PATH/checkpoint` and leverage the eval method as done [here](https://github.com/OATML-Markslab/ProteinNPT/blob/master/scripts/train.py#L298-L303)) but in the meantime you could do it with the current pipeline as follows:
1. Manually create your own seed in your DMS_assay -- let's call it "train_test_split" -- and assign all training points to seed 0 and all test points to seed 1
2. Adjust parameters of pipeline.sh by setting [fold_variable_name](https://github.com/OATML-Markslab/ProteinNPT/blob/master/scripts/pipeline.sh#L14) to "train_test_split" and [test_fold_index](https://github.com/OATML-Markslab/ProteinNPT/blob/master/scripts/pipeline.sh#L15) to 1

The predictions will be stored under `$DATA_PATH/model_predictions`.

Note: the pipeline script will attempt to compute performance on the test seed as if you knew the ground truth for these test points. If the DMS_score values for your test points are truly unknown / missing, the script may complain when computing performance (have not tested this yet) - in which case you may pass in dummy values in your DMS assay csv file for the test points (eg., all zeros).

**Comment by Maxwell-downtown** [comment]
Thanks for your answer, looking forward to the new release.

**Comment by loodvn** [comment]
Hi @Maxwell-downtown - I noticed you've used the repo in the last 7 days so just wanted to give you a heads-up of a change which might impact you.
We've rewritten one of the previous git messages, which impacted the commit history since. 

I've written about how to fix this in this issue: https://github.com/OATML-Markslab/ProteinNPT/issues/11 - please let me know if you run into any problems! Sorry for the inconvenience.

**Comment by pascalnotin** [comment]
Hi @Maxwell-downtown -- as promised, we just pushed an eval script [here](https://github.com/OATML-Markslab/ProteinNPT/blob/master/scripts/eval.sh) to facilitate the prediction on new test sequences. Let us know if any questions!
